perftracker/graphs_analysis.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+
+def pt_comparison_graphs_analyze(data):
+    round_to = 3
+    mse_categories = ('low', 'med', 'high')
+    mse_categories_amount = len(mse_categories)
+    mse_values = []
+    rv = {}
+
+    # coefficient belongs to (0, 1) for little outliers detection
+    # coefficient belongs to (1, +inf) for rough outliers detection
+    # default value is 1.
+    outlier_coefficient = 1.
+
+    for graph_id, graph_data in data.items():
```

Comment:
consider to stick to existing terminology - series[_data] 
technically that data is not necessarily 'graph', it can be barchart etc.